add: spread-the-word.md, link to from README. resolves #158 (#167)
lucavallin/first-issue: README.md

- Create a new pull-request. Please add the link to the issues page of the repository in the PR description. Once the pull request is merged, the changes will be live on [firstissue.dev](https://firstissue.dev/).

## Help us grow!

Navigating open-source projects can be quite overwhelming for begineers and experienced contributers alike. First-issue looks to solve this problem by providing a platform that serves as a starting point for those looking to get started with open-source or those who are looking to get into a new project.

#### How to help

> It's easy! The more people who know about first-issue, the better. There are various ways you can help us grow:

Reviewer: ```suggestion
It's easy! The more people who know about [FirstIssue.dev](http://firstissue.dev), the better. There are various ways you can help us grow:
```

- Contribute to 'Awesome' lists. Everyone uses them. They are community built resources on a wide variety of topics. Find one that relates to getting started with open-source and get first-issue put in there! You could even create one.

> - Blog about us. Have you used first-issue? Do you like what first-issue is doing? Write a blog about your expereince with first issue!

Reviewer: ```suggestion
- Blog about us. Have you ever used [FirstIssue.dev](http://firstissue.dev) and do you like what we are doing? Share your experience on platforms like:
```

>     - Medium

Reviewer: ```suggestion
    - [Medium](http://medium.com)
```

>     - Dev.to

Reviewer: ```suggestion
    - [Dev.to](http://dev.to)
```

    - FreeCodeCamp

>     - ItsFoss

Reviewer: ```suggestion
    - [ItsFoss](https://itsfoss.com)
```

>     - Reddit

Reviewer: ```suggestion
    - [Reddit](https://reddit.com/)
```

>     - Discord

Reviewer: ```suggestion
    - [Discord](https://discord.com)
```

- Reach out to influencers. Twitter and YouTube are full of them. Try and get first-issue mentioned in a video or tweet!
- Make a guide! The best way to learn, is by teaching. This can be a blog or a video. Show what it's like to get started in open-source and use first-issue as a tool for your guide.

> - Be creative! This is a non-comprehensive list of ideas. Once you have written an article, filmed a video, or have gotten us mentioned in a tweet, come back here and add your name along with a link to the media to the list of contributers below!

Reviewer: ```suggestion
- Be creative! This is a non-comprehensive list of ideas. Once you have written an article, recorded a video, or have gotten us mentioned somewhere, come back here and add your name along with a link to the media to the list of contributors below!
```
